src/util/bip324.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_UTIL_BIP324_H
+#define BITCOIN_UTIL_BIP324_H
+
+#include <key.h>
+#include <span.h>
+
+#include <array>
+
+constexpr size_t BIP324_KEY_LEN = 32;
+
+class BIP324Keys
```

Comment:
Not sure if a class for this is desirable in the first place. These keys shouldn't be long-lived in any case (they're only used to initialize the stream ciphers, and then destroyed). That can just as well be done with a simple function call?

Reply:
Done.